src/components/_overlays/MemberDropdown/MemberDropdown.tsx

```diff
@@ -0,0 +1,204 @@
+import { Avatar } from '@/components/Avatar'
+import { ListItem } from '@/components/ListItem'
+import { Text } from '@/components/Text'
+import {
+  SvgActionAddVideo,
+  SvgActionJoyToken,
+  SvgActionMember,
+  SvgActionNewChannel,
+  SvgActionPlay,
+  SvgActionPlus,
+} from '@/components/_icons'
+import { SvgActionSwitchMember } from '@/components/_icons/ActionSwitchMember'
+import { IconWrapper } from '@/components/_icons/IconWrapper'
+import { absoluteRoutes } from '@/config/routes'
+import { AssetType, useAsset } from '@/providers/assets'
+import { useUser } from '@/providers/user'
+import { cVar } from '@/styles'
+
+import {
+  BalanceContainer,
+  BlurredBG,
+  ChannelsSectionTitle,
+  Container,
+  Divider,
+  Filter,
+  MemberInfoContainer,
+  SectionContainer,
+  StyledAvatar,
+  StyledSvgActionChevronL,
+  StyledSvgActionChevronR,
+  SwitchMemberItemListContainer,
+  TjoyContainer,
+} from './MemberDropdown.styles'
+
+export type MemberDropdownProps = { publisher?: boolean }
+
+export const MemberDropdown: React.FC<MemberDropdownProps> = ({ publisher }) => {
+  const [isSwitchingMember, setIsSwitchingMember] = useState(false)
+  const navigate = useNavigate()
+  const { activeChannelId, activeMembership, setActiveUser, memberships } = useUser()
+
+  const hasOneMember = memberships?.length === 1
+
+  const handleAddNewMember = () => {
+    navigate(absoluteRoutes.studio.newMembership())
+  }
+  const handleSwitchMemberMode = () => {
+    setIsSwitchingMember(true)
+  }
+  const handleAddNewChannel = () => {
+    navigate(absoluteRoutes.studio.newChannel())
+  }
+  const handleGoToJoystream = () => {
+    navigate(absoluteRoutes.viewer.index())
+  }
+  const handleGoToStudio = () => {
+    navigate(absoluteRoutes.studio.index())
+  }
+  const handleGoToMyProfile = () => null
+  const handleChannelChange = (channelId: string) => {
+    setActiveUser({ channelId })
+  }
+  const handleMemberChange = (memberId: string) => {
+    setActiveUser({ memberId })
+    setIsSwitchingMember(false)
+  }
+
+  return (
+    <Container>
+      {isSwitchingMember ? (
+        <div>
+          <SwitchMemberItemListContainer>
+            <ListItem
+              onClick={() => setIsSwitchingMember(false)}
+              nodeStart={<StyledSvgActionChevronL />}
+              label="Switch member"
+            />
+          </SwitchMemberItemListContainer>
+
+          <SectionContainer>
+            {memberships?.map((member) => (
+              <ListItem
+                key={member.id}
+                onClick={() => handleMemberChange(member.id)}
+                nodeStart={<Avatar assetUrl={member.avatarUri} />}
+                label={member.handle ?? ''}
+                selected={member.id === activeMembership?.id}
+              />
+            ))}
+            <ListItem
+              nodeStart={<IconWrapper icon={<SvgActionNewChannel />} />}
+              onClick={() => handleAddNewMember()}
+              label={'Add new member...'}
```

Comment:
```suggestion
              label="Add new member..."
```
Ok, we should have some eslint/prettier plugin already for changing that 😅  I think [this one](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-curly-brace-presence.md) should work, but let's do this in another PR.